README.md

```diff
@@ -605,6 +612,38 @@ This returns one row for every `Animal` whose list of aliases contains the value
 for the `$wanted` parameter. Each row contains the matching `Animal`'s name
 in a column named `animal_name`.
 
+#### Constraints and Rules
+- Must be on a property field of list type.
+
+### has_edge_degree
+#### Example Use
+```
+{
+    Animal {
+        name @output(out_name: "animal_name")
+
+        out_Animal_ParentOf @filter(op_name: "has_edge_degree", value: ["$child_count"]) @optional {
+            uuid
```

Comment:
Why do we just have `uuid` here?

Reply:
Because we have to have *something* between the curly braces, to satisfy the GraphQL parser. The `uuid` has no effect, since it's not tagged, output, or filtered on.

Reply:
Yes please! It seems a little out of place.

Reply:
Sounds good, I'll add a note. Thanks!